storeid-wrapper

```diff
@@ -8,13 +8,13 @@ __version__   = "1.2"
 import re
 import sys
 
-HTTPREDIR     = re.compile('^http://httpredir\.debian\.org/.*')
+HTTPREDIR     = re.compile('^http://httpredir\\.debian\\.org/.*')
 
 APT_DOMAIN    = 'http://apt.squid.internal'
-APT_RE_MIRROR = re.compile('^.*/(debian|ubuntu)/(pool/.*)\.(deb|udeb|tar.gz|tar.xz|tar.bz2)$')
+APT_RE_MIRROR = re.compile('^.*/(debian|ubuntu)/(pool/.*)\\.(deb|udeb|tar.gz|tar.xz|tar.bz2)$')
 
 YUM_DOMAIN    = 'http://yum.squid.internal'
-YUM_RE_MIRROR = re.compile('^.*/(x86_64/.*)\.(gz|bz2|xml|srpm|drpm|rpm)$')
+YUM_RE_MIRROR = re.compile('^.*/(x86_64/.*)\\.(gz|bz2|xml|srpm|drpm|rpm)$')
```

Comment:
Can you use the "single-backslash-raw-string" method instead ? (syntax highlighting for regexes are usually better using this)